app/weeklyPicks/page.tsx

```diff
@@ -132,7 +132,7 @@ export default function WeeklyPickForm() {
   };
 
   return (
-    <section className="w-full pt-8">
+    <section className="w-full pt-8" {...form}>
```

Comment:
q: Why are you spreading the form object into the section?

Reply:
I was using this to test my error boundary, forgot to remove it.